frontend/public/components/utils/datetime.ts

```diff
@@ -1,7 +1,7 @@
 import * as _ from 'lodash-es';
 
 // Behaves like moment.js's fromNow
-export const fromNow = (dateTime, now=undefined) => {
+export const fromNow = (dateTime, now=undefined, noSuffix=false) => {
```

Comment:
I'm inclined to do:

```JavaScript 
export const fromNow = (dateTime, now=undefined, options = { noSuffix: false }) => {
}
```
If we ever have to add another option, it makes it easier to keep them named.  Bool args require developers to look up the function to know what the booleans represent (not self documenting).

Do we want to ensure this function remains true to the comment `Behaves like moment.js's fromNow`? Open question, I don't have a strong feeling atm.